source/c_apifunctional.rst

```diff
@@ -4,60 +4,69 @@ API functional documentation
 
 You can acces the Lizard API via “www.{your_organisation}.lizard.net/api/”.
 
-.. image:: /images/api.JPG
+.. image:: /images/c_apifunctional_01.jpg
 
 What is an API?
------------------
+===============
 
 API stands for Application Programming Interface. 
 The API, looking like the picture above, gives back timeseries, rasters, or other data or information. This is depending on the request you do to the API. This request comes from the URL you type in the browser. You can also access the API via another program, and make automatic requests.
 
-
 Basic use API
---------------
+=============
 
-Below we discuss a basic request to the API. More examples and possibilities will be discussed in the :doc:`apitech`.
+Below we discuss a basic request to the API. More examples and possibilities will be discussed further down
 
 The basic url is www.{your_organisation}.lizard.net/api/v3, for example:
```

Comment:
Dit soort dingen kan je het beste tussen dubbele backticks zetten om duidelijker te maken dat het iets computer-achtigs is. Ook wordt www.pietje.nl dan tenminste niet per ongeluk als link gerenderd.
```suggestion
The basic url is ``www.{your_organisation}.lizard.net/api/v3``, for example:
```